keyboards/bastardkb/charybdis/3x5/keymaps/via/readme.md

```diff
@@ -0,0 +1,77 @@
+# Charybdis (3x5) `via` keymap
+
+The Charydbis (3x5) `via` keymap is based on a QWERTY layout with [home row
+mods](https://precondition.github.io/home-row-mods) and [Miryoku-inspired
+layers](https://github.com/manna-harbour/miryoku), and some features and changes
+specific to the Charybdis.
+
+This layout supports RGB matrix. However, due to space constraints on the MCU,
+only a limited number of effect can be enabled at once. Look at the `config.h`
+file and enable your favorite effect.
```

Comment:
```suggestion
The Charydbis (3x5) `via` keymap is based on a QWERTY layout with [home row mods](https://precondition.github.io/home-row-mods) and [Miryoku-inspired layers](https://github.com/manna-harbour/miryoku), and some features and changes specific to the Charybdis.

This layout supports RGB matrix. However, due to space constraints on the MCU, only a limited number of effect can be enabled at once. Look at the `config.h` file and enable your favorite effect.
```

Reply:
Done.  I've removed the extraneous line breaks through out all `readme.md`.